packages/web3-core/types/index.d.ts

```diff
@@ -213,6 +213,7 @@ export interface TransactionReceipt {
     contractAddress?: string;
     cumulativeGasUsed: number;
     gasUsed: number;
+    effectiveGasPrice: number;
```

Comment:
```suggestion
    effectiveGasPrice?: number;
```
I think this should be optional since pre-EIP1559 txs won't have this field, do you agree?

Reply:
Never mind, it looks like clients are returning `gasPrice` for this value regardless